hopsworks-web/yo/app/scripts/controllers/createFeaturegroupCtrl.js

```diff
@@ -102,7 +102,8 @@ angular.module('hopsWorksApp')
                     self.featuresDocWrongValue[i] = 1
                 }
 
-                if (!self.featuregroupName || self.featuregroupName.search(self.hiveRegexp) == -1 || self.featuregroupName.length > 256) {
+                if (!self.featuregroupName || self.featuregroupName.search(self.hiveRegexp) == -1 || self.featuregroupName.length > 256 ||
+                    self.featuregroupName.includes("-")) {
```

Comment:
If you don't want the the - in the names, why not remove it from the regexp instead of doing the check twice?

Reply:
Fixed, my bad. Looked at regexp too quick